config/main.py

```diff
@@ -342,6 +342,70 @@ def load_minigraph():
     _restart_services()
     print "Please note setting loaded from minigraph will be lost after system reboot. To preserve setting, run `config save`."
 
+
+#
+# 'port_channel' group
+#
+@cli.group()
+@click.pass_context
+def port_channel(ctx):
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    ctx.obj = {'db': config_db}
+    pass
+
+@port_channel.command('add')
+@click.argument('port_channel_name', metavar='<port_channel_name>', required=True)
+@click.option('--min-links', default=0, type=int)
+@click.option('--fallback', default='false')
+@click.pass_context
+def add_port_channel(ctx, port_channel_name, min_links, fallback):
+    """Add port channel"""
+    db = ctx.obj['db']
+    fvs = {'admin_status': 'up',
+           'mtu': '9100'}
+    if min_links != 0:
+        fvs['min_links'] = str(min_links)
+    if fallback != 'false':
+        fvs['fallback'] = 'true'
+    db.set_entry('PORTCHANNEL', port_channel_name, fvs)
+
+@port_channel.command('del')
+@click.argument('port_channel_name', metavar='<port_channel_name>', required=True)
+@click.pass_context
+def remove_port_channel(ctx, port_channel_name):
+    """Remove port channel"""
+    db = ctx.obj['db']
+    db.set_entry('PORTCHANNEL', port_channel_name, None)
+
+@port_channel.group('member')
+@click.pass_context
+def port_channel_member(ctx):
+    pass
+
+@port_channel_member.command('add')
+@click.argument('port_channel_name', metavar='<port_channel_name>', required=True)
+@click.argument('port_name', metavar='<port_name>', required=True)
+@click.pass_context
+def add_port_channel_member(ctx, port_channel_name, port_name):
+    """Add member to port channel"""
+    db = ctx.obj['db']
+    db.set_entry('PORTCHANNEL_MEMBER', (port_channel_name, port_name),
+            {'NULL': 'NULL'})
+
+@port_channel_member.command('del')
+@click.argument('port_channel_name', metavar='<port_channel_name>', required=True)
+@click.argument('port_name', metavar='<port_name>', required=True)
+@click.pass_context
+def del_port_channel_member(ctx, port_channel_name, port_name):
+    """Remove member from port channel"""
+    db = ctx.obj['db']
+    db.set_entry('PORTCHANNEL_MEMBER', (port_channel_name, port_name), None)
+    db.set_entry('PORTCHANNEL_MEMBER', port_channel_name + '|' + port_name, None)
+    print port_channel_name
```

Comment:
Suggest to use `click.echo()` instead of `print` for consistency here and the line below. Also, this looks like the output will be confusing as to the result of the operation. Suggest to output a more descriptive message indicating success/failure like, "Member port <port_name> removed from portchannel <port_channel_name>".

Reply:
this was accidentally added here. i just removed it.